src/components/Statuses/Statuses.tsx

```diff
@@ -23,10 +25,45 @@ const Statuses = () => {
     return <div className={styles.base}>Keine Daten vorhanden</div>;
   }
 
+  //sort by date
+  const sortedStatuses: { dateKey: string; statuses: Status[] }[] = [];
+  statuses.forEach((status) => {
+    const departureSchedule = parseSchedule({
+      actual:
+        status.train.origin.departureReal ??
+        status.train.origin.departure ??
+        '',
+      planned: status.train.origin.departurePlanned ?? '',
+    });
+    const date = new Date(departureSchedule.actualValue);
+    const dateKey = `${date.getDate()}.${
+      date.getMonth() + 1
+    }.${date.getFullYear()}`;
+
+    const existingDate = sortedStatuses.find((s) => s.dateKey === dateKey);
+    if (existingDate) {
+      existingDate.statuses.push(status);
+    } else {
+      sortedStatuses.push({
+        dateKey,
+        statuses: [status],
+      });
+    }
+  });
+
   return (
-    <div className={styles.base}>
-      {statuses.map((status) => (
-        <StatusCard key={status.id} status={status} />
+    <div>
+      {sortedStatuses.map((s) => (
+        <div key={s.dateKey} className={styles.base}>
+          <div className={styles.date}>
+            <hr />
+            <span>{s.dateKey}</span>
+            <hr />
+          </div>
+          {s.statuses.map((status) => (
+            <StatusCard key={status.id} status={status} />
+          ))}
+        </div>
       ))}
```

Comment:
Please use a more descriptive name than a single letter for parameters. Something like `group` maybe.

(Preferably we would also sort the `key` attribute behind `className` but that's not so important)